senapy/dosleg/download_from_csv.py

```diff
@@ -1,12 +1,18 @@
-import csv, pprint, requests, json, time, sys
+import csv, pprint, json, time, sys
 
-FILE = sys.argv[1] if len(sys.argv) > 1 else 'dossiers-legislatifs.csv'
+import chardet, requests, slugify
```

Comment:
chardet is neither used nor in requirements ;)

Reply:
too late :smiling_imp: